dmutils/content_loader.py

```diff
@@ -1,32 +1,54 @@
+import collections
 import yaml
 import inflection
 import re
 import os
 
+from .config import convert_to_boolean, convert_to_number
+
 
 class ContentBuilder(object):
+    """An ordered set of sections each made up of one or more questions.
+
+    Usage::
+
+        >>> content = ContentBuilder(sections)
```

Comment:
Running `py.test` with `--doctest-modules` picks this up as a doctest which fails. Making this work as a test probably wouldn' t make a useful usage example, so I think we should just drop the `>>>` and explain the return value separately.